### PR TITLE
Add tariff information to channel available in sites

### PR DESCRIPTION
**Pull request by iconnor**
While waiting for my smart meter, I noticed my cost was more than a friend's. When I dug into this, I found that my site's tariff was higher. This information is available on the site's channel, so I have updated the code so this can also be seen through the Python wrapper to the API.

image

### CODE REVIEW

**Review by madpilot — APPROVED** [review]

**Comment by iconnor** [comment]
Did you want me to update the tests to use more modern pythons? 3.6 should probably go and now start at 3.7 and cover 3.9, 3.10 and 3.11?

**Comment by madpilot** [comment]
Yes, please - and if you can update the test to check for the new attribute that would be ace.

**Comment by iconnor** [comment]
Locally the tests now pass:
```bash
pytest
================================ test session starts ================================
platform linux -- Python 3.7.12, pytest-7.1.2, pluggy-0.13.1
rootdir: /home/iconnor/workspace/iconnor/amberelectric.py
plugins: mock-3.8.2, doctestplus-0.12.0, astropy-0.10.0, remotedata-0.3.3, hypothesis-6.36.0, astropy-header-0.1.2, filter-subpackage-0.1.1, arraydiff-0.3, openfiles-0.5.0, cov-3.0.0, anyio-3.6.1
collected 7 items                                                                   

test/test_amber_api.py .......                                                [100%]

================================= 7 passed in 0.06s =================================

```

**Comment by madpilot** [comment]
No pandas please, there is zero reason for that to be in this package.

**Comment by madpilot** [comment]
And how do timezones break things?

File: amberelectric/api/amber_api.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
 from typing import List, Optional, Union
 import json
-import dateutil.parser
+from dateutil import tz, parser
 from datetime import date
+import pandas as pd
```
Comment (madpilot) [inline amberelectric/api/amber_api.py:5]: This doesn't belong in this API.

File: amberelectric/api/amber_api.py [hunk 2]
```diff
@@ -206,3 +208,24 @@ def get_usage(self, site_id: str, start_date: date, end_date: date, **kwargs) ->
             return parse_intervals(json.loads(response.data.decode("utf-8")))
         else:
             raise ApiException(response.status, response.reason, response)
+        
+    def get_usage_dateframe(self, site_id: str, start_date: date, end_date: date, **kwargs):
```
Comment (madpilot) [inline amberelectric/api/amber_api.py:212]: Doesn't belong in this API.

File: requirements.txt [hunk 3]
```diff
@@ -1,3 +1,4 @@
 python_dateutil >= 2.5.3
 setuptools >= 21.0.0
 urllib3 >= 1.25.3
+pandas >= 1.1.4
```
Comment (madpilot) [inline requirements.txt:4]: Please remove

**Review by madpilot — CHANGES_REQUESTED** [review]

**Comment by madpilot** [comment]
Superseeded by https://github.com/madpilot/amberelectric.py/pull/9

**Comment by iconnor** [comment]
Sorry about neglecting this PR; I switched to LocalVolts to get five-minute windows.

**Pull request closed without merging**